packages/eas-cli/src/graphql/mutations/EnvironmentVariableMutation.ts

```diff
@@ -24,6 +24,7 @@ type CreateVariableArgs = {
   visibility: EnvironmentVariableVisibility;
   environments: EnvironmentVariableEnvironment[];
   type: EnvironmentSecretType;
+  isGlobal?: boolean;
```

Comment:
shouldn't you regenrate gql schema?

Reply:
I don't think so, `isGlobal` was added (but not used here) long time ago. It is defined in `CreateSharedEnvironmentVariableInput` and in `graphql.schema.json`